File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.js
const fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString().trim().split("\n");

input.forEach((row, i) => {
  arr = row.split(" ");
  avg_arr = (arr.reduce((acc, cur) => acc + cur * 1, 0) - arr[0] * 1) / arr[0];
  let count = 0;
  for (i = 1; i < arr.length; i++) {
    if (arr[i] > avg_arr) count += 1;
  }
  if (i != 1) {
    console.log(((count * 100) / arr[0]).toFixed(3) + "%");
  }
});

File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.py
import sys

def is_over(n):
    return n>avg_row
        
arr=[list(map(int,input().split())) for x in range(int(input()))]

for row in arr:
    avg_row = (sum(row[1:]))/row[0]
    over_list = list(filter(is_over,row[1:]))
    print(f"{round((len(over_list))/row[0]*100,3)}{'%'}")

